doc/supportedPlatformVersions.md

```diff
@@ -65,6 +66,7 @@
 - 7.0.304
 - 7.0.305
 - 7.0.306
+- 8.0.100-rc.1.23463.5
```

Comment:
do we release dotnet 8 other than `bookworm` flavor? if it is then why we are including the dotnet 8 version in all the flavors (in this file only)?